clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1826,8 +1823,10 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
   O << "// This is auto-generated SYCL integration header.\n";
   O << "\n";
 
+  O << "#include <cstddef>\n";
```

Comment:
I don't think we need these changes to the generated header (this and the next change). Because there is no promise that std types work as lambda name.

Reply:
@premanandrao, could you address this comment, please?

Reply:
@bader , @jtmott-intel is working on it.